### ISSUE TITLE
Feature Request/Permintaan fitur baru

### ISSUE THREAD

**Issue by FaizBastomi** [FaizBastomi/wbot#1]
Silahkan bagi kalian yang punya ide mau ditambah fitur apa, biar makin banyak,
tentu kalian juga bisa membuat **Pull Request** nanti akan saya review :D
Kalian juga bisa buat issue baru dan beri label `feature request` agar cepat di notice.

Please, for those of you who have an idea what features to add, so that there are more features in this bot,
Of course you can also make a **Pull Request** I will review it later :D
You can also create a new issue and label it `feature request` so that it gets noticed quickly.

----
### To do
- [x] Limit & Premium Only (BETA)
- [x] broadcast
- [x] Weebs Related Features
---
Terkait nomor terkena ban atau di blokir oleh WhatsApp karena sering melakukan "kick/menendang" anggota dari grup, ini bukanlah kesalahan dari source code di repository ini, melainkan memang sistem deteksi WhatsApp yang sangat sensitif dengan ha ini. Mohon untuk tidak melakukan spam di tab **issue** repo ini. Terima kasih.

Regarding the number being banned or blocked by WhatsApp because often **kicks** members out of the group, this is not the fault of the source code in this repository, but the WhatsApp detection system which is very sensitive to this. Please don't spam the **issue** tab of this repo. Thank you.

**Comment by OyenDCraft** [comment]
Caranya mendapatkan code qr gimana?

**Comment by FaizBastomi** [comment]
> Caranya mendapatkan code qr gimana?

@OyenDCraft bisa di perjelas tidak pertanyaan nya?

**Comment by natxdixie** [comment]
> Silahkan bagi kalian yang punya ide mau ditambah fitur apa, biar makin banyak, tentu kalian juga bisa membuat **Pull Request** nanti akan saya review :D Kalian juga bisa buat issue baru dan beri label `feature request` agar cepat di notice.
> 
> Please, for those of you who have an idea what features to add, so that there are more features in this bot, Of course you can also make a **Pull Request** I will review it later :D You can also create a new issue and label it `feature request` so that it gets noticed quickly.
> 
> ### To do
> * [ ]  Weebs Related Features

req : support heroku

**Comment by FaizBastomi** [comment]
> req : support heroku

@Natx-C , okay

**Comment by natxdixie** [comment]
> > req : support heroku
> 
> @Natx-C , okay

siap om

**Comment by dmibots** [comment]
Please Update Support to okteto.com

**Comment by RrZNBot** [comment]
please add limit and game

**Comment by RrZNBot** [comment]
Apakah bisa ganti welcomenya jadi welcome Canvas button 5?
![IMG_20220404_142924](https://user-images.githubusercontent.com/74750827/161494790-dd3836be-dff5-46ee-ad45-c83675087e18.jpg)

**Comment by FaizBastomi** [comment]
@RrZNBot 
> please add limit and game

kapan2 nanti saya tambahkan

> Apakah bisa ganti welcomenya jadi welcome Canvas button 5?

edit saja di `event/group_event.js` bisa Anda kustomisasi sesuka hati

---
@dmibots 
> Please Update Support to okteto.com

I will try to support the platform

**Comment by seiko-api** [comment]
Assalamualaikum bang tambah broadcast sama owner, isOwner Begitu.

**Comment by FaizBastomi** [comment]
@seiko-api 
> Assalamualaikum bang tambah broadcast sama owner, isOwner Begitu.

oke, broadcast

**Comment by FaizBastomi** [comment]
@seiko-api 
> Assalamualaikum bang tambah broadcast sama owner, isOwner Begitu.

sudah ditambahkan, bila ada eror buat saja _issue_ baru. c9e83e7e7db790f7895fecafb27d8c34ba239b2f

**Comment by FaizBastomi** [comment]
@RrZNBot 
> please add limit and game

untuk game saya tidak bisa,
saya ingin repo ini tetap simpel,
untuk limit saya sedang memikirkannya.

**Comment by dmibot** [comment]
bang tambahin support buat API zenzapi.xyz ya tq, karena disana ada update info buat jadwaltv, info gempa dan buat scrape AlQuran, sangat bermanfaat bg

**Comment by FaizBastomi** [comment]
@dmibot 
> bang tambahin support buat API zenzapi.xyz ya tq, karena disana ada update info buat jadwaltv, info gempa dan buat scrape AlQuran, sangat bermanfaat bg

oke, ntar siang

**Comment by FaizBastomi** [comment]
@Natx-C 
> req : support heroku

untuk heroku sudah ada di branch [heroku](https://github.com/FaizBastomi/wbot/tree/heroku)
silahkan di cek

**Comment by natxdixie** [comment]
> @Natx-C
> 
> > req : support heroku
> 
> untuk heroku sudah ada di branch [heroku](https://github.com/FaizBastomi/wbot/tree/heroku) silahkan di cek

makasih bang request nya :>

**Comment by iamriz7** [comment]
req : fitur todo list

**Comment by FaizBastomi** [comment]
@iamriz7 
> req : fitur todo list

fitur yang bagaimana tuh?

**Comment by iamriz7** [comment]
> @iamriz7
> 
> > req : fitur todo list
> 
> fitur yang bagaimana tuh?

![image](https://user-images.githubusercontent.com/66578450/166910371-24edd93b-6248-40b8-9a01-8caabd8c5002.png)

**Comment by ninoroes07** [comment]
Request sistem sewa bot, anonymous chat, pinterest, google search (image or query), sticker with own watermark, ability to set sticker as command alternative.

Kebanyakan ngak ya request fiturnya:v semoga bisa

**Comment by DavNaIl** [comment]
Request fitur simple aja, fitur auto read & auto delete dong. Soalnya bot tanpa auto read berat nge cache pesan unreadnya, mungkin dibikin interval setiap beberapa jam auto read & deletenya. Sekian

**Comment by itsreimau** [comment]
Di mana saya bisa mendapatkan igCookies?

**Comment by ndhet** [comment]
> Silahkan bagi kalian yang punya ide mau ditambah fitur apa, biar makin banyak, tentu kalian juga bisa membuat **Pull Request** nanti akan saya review :D Kalian juga bisa buat issue baru dan beri label `feature request` agar cepat di notice.
> 
> Please, for those of you who have an idea what features to add, so that there are more features in this bot, Of course you can also make a **Pull Request** I will review it later :D You can also create a new issue and label it `feature request` so that it gets noticed quickly.
> 
> ### To do
> * [x]  Limit & Premium Only (BETA)
> * [x]  broadcast
> * [x]  Weebs Related Features
> 
> Terkait nomor terkena ban atau di blokir oleh WhatsApp karena sering melakukan "kick/menendang" anggota dari grup, ini bukanlah kesalahan dari source code di repository ini, melainkan memang sistem deteksi WhatsApp yang sangat sensitif dengan ha ini. Mohon untuk tidak melakukan spam di tab **issue** repo ini. Terima kasih.
> 
> Regarding the number being banned or blocked by WhatsApp because often **kicks** members out of the group, this is not the fault of the source code in this repository, but the WhatsApp detection system which is very sensitive to this. Please don't spam the **issue** tab of this repo. Thank you.

Anu bang bug WA update skrng ga bisa kirim URLBUTTON 
,MAKASIH

**Comment by FaizBastomi** [comment]
@henceutbeey 
> Anu bang bug WA update skrng ga bisa kirim URLBUTTON
,MAKASIH

[comment/#33](https://github.com/FaizBastomi/wbot/issues/33#issuecomment-1229570885)

**Comment by ndhet** [comment]
> Silahkan bagi kalian yang punya ide mau ditambah fitur apa, biar makin banyak, tentu kalian juga bisa membuat **Pull Request** nanti akan saya review :D Kalian juga bisa buat issue baru dan beri label `feature request` agar cepat di notice.
> 
> Please, for those of you who have an idea what features to add, so that there are more features in this bot, Of course you can also make a **Pull Request** I will review it later :D You can also create a new issue and label it `feature request` so that it gets noticed quickly.
> 
> ### To do
> * [x]  Limit & Premium Only (BETA)
> * [x]  broadcast
> * [x]  Weebs Related Features
> 
> Terkait nomor terkena ban atau di blokir oleh WhatsApp karena sering melakukan "kick/menendang" anggota dari grup, ini bukanlah kesalahan dari source code di repository ini, melainkan memang sistem deteksi WhatsApp yang sangat sensitif dengan ha ini. Mohon untuk tidak melakukan spam di tab **issue** repo ini. Terima kasih.
> 
> Regarding the number being banned or blocked by WhatsApp because often **kicks** members out of the group, this is not the fault of the source code in this repository, but the WhatsApp detection system which is very sensitive to this. Please don't spam the **issue** tab of this repo. Thank you.

Bg tambagin fitur remove message as admin hehehe keren
